openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.tsx

```diff
@@ -233,7 +233,7 @@ const ProfilerTab: React.FC<ProfilerTabProps> = ({
             <Col span={8}>
               <Statistic
                 title="Data type"
-                value={activeColumnDetails.dataTypeDisplay || ''}
+                value={activeColumnDetails.dataType || ''}
```

Comment:
showing `dataType` instead of `dataTypeDisplay`, Is this intended?

Reply:
Yes